docs/index.md

```diff
@@ -29,4 +29,38 @@ Percona XtraBackup {{vers}} does not support making backups of databases
 created in versions before {{vers}} of MySQL, Percona Server for MySQL or
 Percona XtraDB Cluster.
 
+Taking your backup with Percona XtraBackup is easy. Follow our documentation guides, and you’ll be set up in quickly.
+
+<div data-grid markdown><div data-banner markdown>
+
+## :material-frequently-asked-questions: How Percona Xtrabackup works { .title }
+
+Learn about how Percona Xtrabackup works.
+
+[How Percona XtraBackup works :material-arrow-right:](how-xtrabackup-works.md){.md-button}
+
```

Comment:
This one should be closed with </div><div data-banner markdown>

Reply:
done